build-tools/packages/build-tools/src/fluidBuild/tasks/leaf/leafTask.ts

```diff
@@ -47,7 +47,7 @@ export abstract class LeafTask extends Task {
     }
 
     public get isDisabled() {
-        const isLintTask = this.executable === "eslint" || this.executable === "tsfmt";
+        const isLintTask = this.executable === "eslint";
```

Comment:
@jikim-msft These are the fluid-build changes I was talking about. This file and the related change in taskFactory.ts. If you don't revert these changes then the next release of fluid-build (which many folks use to build the repo) won't parse tsfmt tasks correctly. This may mean that fluid-build will stop working on older branches where the tsfmt task still exists. So I think it's safest to leave fluid-build able to parse tsfmt tasks for the time being.